src/EventHandler/AbstractMessage.php

```diff
@@ -208,4 +209,48 @@ public function reply(
             updateStickersetsOrder: $updateStickersetsOrder
         );
     }
+
+    /**
+     * Reply inlineBotResult to message
+     *
+     * @param BigInteger $queryId Query ID from <a href='https://docs.madelineproto.xyz/API_docs/methods/messages.getInlineBotResults.html'>messages.getInlineBotResults</a>
```

Comment:
This should be an int

Reply:
In the doc its Long so thats not good idea to limit it to int !

Reply:
MadelineProto only supports 64-bit builds of PHP, and integers on 64-bit PHP are 64-bit long, so integers must be used.